stroyprombeton/tests/tests_views.py

```diff
@@ -571,6 +571,20 @@ def test_admin_autocomplete_no_results(self):
         self.assertFalse(json_to_dict(response))
         self.assertNotContains(response, term)
 
+    def test_autocomplete_has_only_active(self):
+        """Autocomplete items should contain only active products."""
+        test_product = models.Product.objects.first()
+        test_product.page.is_active = False
+        test_product.page.save()
+
+        url = f'{reverse("autocomplete")}?term={self.TERM}'
```

Comment:
`self.TERM` is too generic for the test case, because `test_product` can be cut off from a final response data in this way: `autocomplete_data[:MAX_COUNT]`

So we should precise our query: `term=(test_product.name[:n] or test_product.name)`